platform/product_packages.mk

```diff
@@ -40,6 +40,10 @@ PRODUCT_PACKAGES += \
 PRODUCT_PACKAGES += \
     Snap
 
+### CHARGER
+PRODUCT_PACKAGES += \
+     lineage_charger_res_images
+
```

Comment:
Isn't that obsolete if we have our own animation?

Reply:
Yes, forgot to remove it